backend/src/main/java/wooteco/prolog/studylog/ui/ReportController.java

```diff
@@ -0,0 +1,65 @@
+package wooteco.prolog.studylog.ui;
+
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort.Direction;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import wooteco.prolog.login.domain.AuthMemberPrincipal;
+import wooteco.prolog.member.domain.Member;
+import wooteco.prolog.studylog.application.ReportService;
+import wooteco.prolog.studylog.application.dto.report.request.ReportRequest;
+import wooteco.prolog.studylog.application.dto.report.response.ReportResponse;
+
+@RestController
+public class ReportController {
+
+    private final ReportService reportService;
+
+    public ReportController(ReportService reportService) {
+        this.reportService = reportService;
+    }
+
+    @GetMapping("/{username}/}reports")
```

Comment:
중괄호 닫는 부분이 하나 빠져야 하는건가요?
```suggestion
    @GetMapping("/{username}/reports")
```

Reply:
아래의 두 표현은 비슷하지만 다른 의미를 지니고 있습니다.
- /{username}/reports: 특정 사용자의 리포트를 가리킴
- /reports?member={username}: 리포트 중 작성자가 특정 사용자인 리포트를 가리킴

저는 두번째 표현을 써보는게 어떨지 추천하고싶은데요
이유는 크게 다르진 않지만 나중에 필터링을 하거나 리포트 목록을 조회하는 경우 조건이 추가적으로 생길 때 좀 더 유연하게 확장할 수 있을 것 같습니다.

Reply:
좋은것 같습니다! queryParam을 사용하는게 더욱 유연하게 느껴지네요!!!

저 오탈자는... 있는데 어떻게 정상 작동이 됐던거지..? 스프링... 갓...

Reply:
일단 두가지 API를 모두 유지시키고 이전 버전은 Deprecated 시켜놨습니다.